lib/portlayer/exec/commit.go

```diff
@@ -79,7 +79,7 @@ func Commit(ctx context.Context, sess *session.Session, h *Handle, waitTime *int
 		}
 
 		if err != nil {
-			log.Errorf("Something failed. Spec was %+v", *h.Spec.Spec())
+			log.Errorf("Waiting for operations to complete failed: %s. Spec was %+v", err, *h.Spec.Spec())
```

Comment:
perhaps `An error occurred while waiting for a creation operation to complete...` :-) that way someone might be able to zip right to this section of the code for commit when they see this error.